google/cloud/storage/internal/rest_client.cc

```diff
@@ -891,453 +891,147 @@ StatusOr<QueryResumableUploadResponse> RestClient::UploadChunk(
 
 StatusOr<ListBucketAclResponse> RestClient::ListBucketAcl(
     ListBucketAclRequest const& request) {
-  auto const& current = CurrentOptions();
-  RestRequestBuilder builder(
-      absl::StrCat("storage/", current.get<TargetApiVersionOption>(), "/b/",
-                   request.bucket_name(), "/acl"));
-  auto auth = AddAuthorizationHeader(current, builder);
-  if (!auth.ok()) return auth;
-  request.AddOptionsToHttpRequest(builder);
-  return ParseFromRestResponse<ListBucketAclResponse>(
-      storage_rest_client_->Get(std::move(builder).BuildRequest()));
+  return curl_client_->ListBucketAcl(request);
```

Comment:
I do not think @scotthart will appreciate this change

Reply:
Ooops, bad merge, fixed.